Web/starter_project_vanguard/components/profile/PersonalInfo.tsx

```diff
@@ -0,0 +1,120 @@
+import Image  from 'next/image'
+import React from 'react'
+
+import {User} from '@/types/user-profile'
+import profile from 'public/images/profile-photo.png'
```

Comment:
[nit] i think you should stick to the profilePhoto name for the variable too to stay consistent